portfolio/src/main/webapp/index.html

```diff
@@ -134,7 +136,15 @@ <h2>Misc</h2>
     </div>
     <!--Comments-->
     <div class="comment_background">
-      <div id="comments_sec"></div>
+      <div id="comments_sec">
+        <h2>Leave a Comment!</h2>
+        <form action="/comments" method="POST">
+          <textarea name="comments-input"class="form-control" rows="6";></textarea>
+          <br><input type="submit">
+        </form>
+        <br><br>
```

Comment:
Is this meant to be two line breaks in a row? If so, put them on separate lines. Otherwise, just `<br>` works since it's self-closing.

See also my comment about `<br>` above: I think you can probably omit it but it's not a high priority.